docs/source/raft_ann_benchmarks.md

```diff
@@ -152,6 +152,11 @@ options:
   -f, --force           re-run algorithms even if their results already exist (default: False)
   -m SEARCH_MODE, --search-mode SEARCH_MODE
                         run search in 'latency' (measure individual batches) or 'throughput' (pipeline batches and measure end-to-end) mode (default: throughput)
+  -t SEARCH_THREADS, --search-threads SEARCH_THREADS
+                        specify the number threads to use for throughput benchmark. Single value or a pair of min and max separated by ':'. Example --threads=1:4. Power of 2 values between 'min' and 'max' will be used. If only 'min' is
```

Comment:
Example should be `--search-threads`

Reply:
ooh good catch. This was copied from the c++ binary.

Reply:
I guess the script's help section will also have to be updated?

Reply:
Yeah, I updated that as well